source/extensions/omni.isaac.lab/omni/isaac/lab/sim/converters/mesh_converter_cfg.py

```diff
@@ -42,3 +42,9 @@ class MeshConverterCfg(AssetConverterBaseCfg):
 
     "none" causes no collision mesh to be added.
     """
+
+    rotation: tuple[float, float, float, float] | None = None
```

Comment:
NIT: Maybe shoulw specify the default to None means whatever is the orientation and scale in the mesh file?